open_instruct/utils.py

```diff
@@ -278,6 +286,13 @@ def get_datasets(
                 and "messages" not in dataset.column_names
             ):
                 dataset = dataset.map(convert_metamath_qa_to_messages, num_proc=10)
+            elif (
```

Comment:
where is this coming from?

Reply:
this was the original point of this branch, adding support for loading our rejection sampling datasets -- they're both small changes so I didn't bother splitting them. I already confirmed this works + it fits with our general schema for loading non-messages-oriented datasets

Reply:
would be good to document this in PR description :)

Reply:
was just about to :) done!